operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -1354,7 +1360,7 @@ function wdtRotateAndCopyLogFile() {
   testLogFileRotate "${WDT_OUTPUT_DIR}/${logFileName}"
```

Comment:
Should this be removed since logFileRotate already called it?

Reply:
Removed.